JS.md

```diff
@@ -243,8 +242,30 @@ of `exports`. Let `moduleNamespace` be the result of calling
 [`ModuleNamespaceCreate(moduleRecord, exportStrings)`](http://tc39.github.io/ecma262/#sec-modulenamespacecreate).
 Set `moduleRecord.[[Namespace]]` to `moduleNamespace`.
 
-Return a new `WebAssembly.Instance` object setting `[[Instance]]` to `instance`
-and `exports` to `moduleNamespace`.
+Let `instanceObject` be a new `WebAssembly.Instance` object setting
+`[[Instance]]` to `instance` and `exports` to `moduleNamespace`.
+
+If any of the elements of an [Elements section](Modules.md#elements-section)
+refer to an imported function which is not an
+[Exported Function Exotic Object](#exported-function-exotic-objects), throw a
+[`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror).
```

Comment:
This is just mirroring the MVP constraints added in #775 that tables only contain WebAssembly functions.